iphone/cli/commands/_buildModule.js

```diff
@@ -476,6 +483,25 @@ iOSModuleBuilder.prototype.buildModule = function buildModule(next) {
 			args.push('SUPPORTS_MACCATALYST=YES');
 		}
 
+		if (target === 'iphonesimulator') {
+			// Search for third party framewrok included in module. If found any, exclude arm64 fromsimulator build.
+			// Assumption is that simulator arm64  architecture can only be supported via .xcframework.
+			const frameworkPattern = /([^/]+)\.framework$/;
+			let frameworksPath = path.join(this.projectDir, 'platform');
+			let legacyFrameworks = new Set();
+
+			fs.readdirSync(frameworksPath).forEach(function (filename) {
+				let possibleFrameworkPath = path.join(frameworksPath, filename);
+				if (frameworkPattern.test(possibleFrameworkPath)) {
```

Comment:
This was probably stolen from `frameworks.js` code, but I don't think we need a regex here or to build the path, just check for `filename.endsWith('.framework')`